### PR TITLE
☄️ Handle warning in add proposal modal

### LINKED ISSUE #833: Proposal warning
@[figma](https://www.figma.com/file/pgNl83wgAub186krVkdiuo/Proposals?node-id=15%3A4385)

Add a warning window:
- display warning message
- user must check checkbox to go further

### PR DESCRIPTION
**Pull request by DevNikita16**
Closes #833

### CODE REVIEW

File: packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalWarningModal.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  return (
+    <Modal modalSize="m" modalHeight="s" onClose={hideModal}>
+      <ModalHeader onClick={hideModal} title="Caution" icon={<AlertSymbol />} />
+      <ModalBody>
+        <TextMedium margin="s">
+          Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim
+          velit mollit. Exercitation veniam consequat sunt nostrud amet.
+        </TextMedium>
+        <TextInfo>
```
Comment (jodator) [inline packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalWarningModal.tsx:29]: this should be `Info` component from `'@/common/components/Info'`

Comment (jodator) [reply packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalWarningModal.tsx:29]: I see also that the title is not needed here, so you can make it optional for that component as well.

**Review by jodator — CHANGES_REQUESTED** [review]
Nice 🚀 

The only thing is to use `Info` component for that  "grey" box on the wireframes.

File: packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalWarningModal.tsx (hunk 1 above)
Comment (DevNikita16) [reply packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalWarningModal.tsx]: ✅

**Review by jodator — APPROVED** [review]

**Pull request merged**